### ISSUE TITLE
have entity group members trigger update

### ISSUE THREAD

**Issue by Mariusthvdb** [custom-cards/button-card#392]
**Is your feature request related to a problem? Please describe.**

I need a hack to have the button update on group members when entity is a group
**Describe the solution you'd like**

have entity automatically check for group members updates, like jinja `expand('group.members')`

**Describe alternatives you've considered**

now using an automation with all entities verbosely listed and have that update the button. Or alternatively, list the entities in the `triggers_update:` field. This is rather tedious with larger groups (eg all lights) and needs maintenance on change.

**Additional context**

```
  - type: custom:button-card
    entity: group.outside_lights
    triggers_update: input_boolean.run_lights_summary
    template: button_title_outdoors
    label: > #Outside lights
      [[[
      var i;
      var entities = entity.attributes.entity_id;
      var count = 0;
      for (i = 0; i < entities.length; i++) {
        var state = states[entities[i]].state;
        if (state == 'on') {count += 1;}
       }
      if (count == entities.length) return 'All ' + entities.length + ' Outside lights on';
      if (count == 0) return 'No Outside lights on';
      return 'Outside Lights on: ' + count + ' of ' + entities.length;
      ]]]
```

see: https://community.home-assistant.io/t/lovelace-button-card/65981/4729?u=mariusthvdb

one caveat: users might no need the members for triggering, and truly only use the group state. In which case this FR would also include an option?
Maybe `update_group: true`, and setting to default 'False' to prevent unwanted updating?

secondly: have this group members check be recursive. See [this for my personal quest](https://community.home-assistant.io/t/need-help-selecting-entities-in-grouped-groups-in-python-script/234389/9?u=mariusthvdb) creating a function and have a JS template count the group members......

thanks for considering!

**Comment by RomRider** [comment]
Would you mind trying the latest beta version which implements this feature and report back please? (3.4.0-2)

**Comment by Mariusthvdb** [comment]
can confirm this to be working perfectly:
```
  - type: custom:button-card
    entity: group.all_inside_lights
    template: button_title_counter
    group_expand: true
#    triggers_update: input_boolean.run_lights_summary
    label: >
      [[[
        if (!entity) return 0;
        function loop(list, stats = {total: 0, enabled : 0}) {
          list.forEach(child_entity => {
            if (states[child_entity])
              if (child_entity.split('.')[0] === "group") {
                stats = loop(states[child_entity].attributes.entity_id, stats);
              } else {
                stats.total += 1;
                if (["open", "on"].includes(states[child_entity].state))
                  stats.enabled += 1;
              }
          });
          return stats;
        }
        let stats = loop(entity.attributes.entity_id);

        if (stats.enabled == stats.total) return 'All ' + stats.enabled + ' lights on';
        if (stats.enabled == 0) return 'No lights on';
        return 'Lights on: ' + stats.enabled + ' of ' + stats.total;

        ]]]
```

thank you very much!!

**Comment by RomRider** [comment]
Feature released in 3.4.0, closing
